src/components/EventList/components/EventItem/index.jsx

```diff
@@ -1,10 +1,14 @@
-import { CardContent, Collapse, Grid, Typography } from '@mui/material';
+import { Button, CardContent, Collapse, Grid, Typography } from '@mui/material';
 import { useState } from 'react';
 import styles from './EventItem.module.css';
+import { AddToCalendarButton } from 'add-to-calendar-button-react';
```

Comment:
Is this a new package? I don't see a new listing in `package.json`. Am I missing something?

Reply:
Yes! This is the only package I added

Reply:
See my later comment concerning this.